external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -185,7 +178,9 @@ private long doSeek(TopicPartition tp, OffsetAndMetadata committedOffset) {
     }
 
     private Collection<TopicPartition> toArrayList(final TopicPartition tp) {
-        return new ArrayList<TopicPartition>(1){{add(tp);}};
+        ArrayList<TopicPartition> ret = new ArrayList<>(1);
```

Comment:
Nit: Unless you need to modify the return value later, you can just return Collections.singleton(tp)